Patch to allow us to specify an alternate tidy binary when running tests.

This patch should be sent upstream.

--- tidy-1.0.0/test/testone.sh.orig	2015-07-31 11:20:28.840693501 -0700
+++ tidy-1.0.0/test/testone.sh	2015-07-31 11:20:59.008447677 -0700
@@ -24,7 +24,7 @@
 
 TESTNO=$1
 EXPECTED=$2
-TIDY=../console/tidy
+TIDY="${TIDY_BIN:-../console/tidy}"
 INFILES=./input/in_${TESTNO}.*ml
 CFGFILE=./input/cfg_${TESTNO}.txt
 
